SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -698,7 +764,8 @@ Worker_CommandRequest USpatialSender::CreateRPCCommandRequest(UObject* TargetObj
 		}
 	}
 
-	WriteRpcPayload(RequestObject, TargetObjectRef.Offset, CommandIndex, PayloadWriter);
+	RPCPayload Payload(TargetObjectRef.Offset, CommandIndex, TArray<uint8>(PayloadWriter.GetData(), PayloadWriter.GetNumBytes()));
```

Comment:
We do one unnecessary copy of the payload as a result of this change. Maybe `WriteToSchemaObject` could be a static function that takes the offset, index, data, and byte count?